torchci/pages/api/drci/drci.ts

```diff
@@ -194,41 +194,53 @@ export function constructResultsComment(
     const headerPrefix = `## `
     const pendingIcon = `:hourglass_flowing_sand:`
     const successIcon = `:white_check_mark:`
-    const noneFailing = ` No Failures`;
-    const someFailing = `## :x: ${failing} Failures`;
-    const somePending = `, ${pending} Pending`;
-
-    const hasFailing = failing > 0;
-    const hasPending = pending > 0;
-    if (!hasFailing) {
-        output += headerPrefix
-        if (hasPending) {
-            output += pendingIcon
-        } else {
-            output += successIcon
-        }
-
-        output += noneFailing;
-
-        if (hasPending) {
-            output += somePending;
-        }
-
-        output += `\nAs of commit ${sha}:`;
-        output += `\n:green_heart: Looks good so far! There are no failures yet. :green_heart:`;
+    const failuresIcon = `:x:`
+    const noneFailing = `No Failures`
+    const significantFailures = `${failedJobs.length} Significant Failures`
+    const unrelatedFailures = `${flakyJobs.length + brokenTrunkJobs.length} Unrelated Failures`
+    const pendingJobs = `${pending} Pending`
+
+    const hasAnyFailing = failing > 0
+    const hasSignificantFailures = failedJobs.length > 0
+    const hasPending = pending > 0
+    const hasUnrelatedFailures = flakyJobs.length + brokenTrunkJobs.length
+    
+    let icon = ''
+    if (hasSignificantFailures) {
+      icon = failuresIcon
+    } else if (hasPending) {
+      icon = pendingIcon
     } else {
-        output += someFailing;
-        if (hasPending) {
-            output += somePending;
-        }
-        output += `\nAs of commit ${sha}:`;
-        output += constructResultsJobsSections(
-          hud_pr_url,
-          "NEW FAILURES",
-          "The following jobs have failed",
-          failedJobs,
-        );
+      icon = successIcon
+    }
+
+    let title_messages = []
+    if (hasSignificantFailures) {
+      title_messages.push(significantFailures)
+    }
+    if (!hasAnyFailing) {
+      title_messages.push(noneFailing)
+    }
+    if (hasPending) {
+      title_messages.push(pendingJobs)
+    }
+    if (hasUnrelatedFailures){
+      title_messages.push(unrelatedFailures)
     }
+
+    let title = headerPrefix + icon + ' ' + title_messages.join(', ')
+    output += title
+    output += `\nAs of commit ${sha}:`
+
+    if (!hasAnyFailing) {
+      output += `\n:green_heart: Looks good so far! There are no failures yet. :green_heart:`;
+    }
+    output += constructResultsJobsSections(
```

Comment:
Is this possible to collapse the BROKEN TRUNK and FLAKY sections to hide them a bit better on Dr. CI?

Reply:
good idea